Gemfile

```diff
@@ -3,7 +3,9 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in amazon_ssa_support.gemspec
 gemspec
 
+gem "log_decorator", ">0", :require => 'log_decorator', :git => "https://github.com/ManageIQ/log_decorator.git", :branch => "master"
 gem "manageiq-gems-pending", ">0", :require => 'manageiq-gems-pending', :git => "https://github.com/ManageIQ/manageiq-gems-pending.git", :branch => "master"
+gem "manageiq-smartstate", ">0", :require => 'manageiq-smartstate', :git => "https://github.com/ManageIQ/manageiq-smartstate.git", :branch => "master"
```

Comment:
`manageiq-smartstate` is a published gem, so you should remove the `:git =>` portion.